.github/workflows/release.yml

```diff
@@ -82,20 +82,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [linux, macos, win-msvc]
+        build: [linux, macos-amd64, macos-arm64, win-msvc]
         include:
         - build: linux
           os: ubuntu-20.04
           rust: stable
           target: x86_64-unknown-linux-gnu
           asset_name: boa-linux-amd64
           binary_name: boa
-        - build: macos
+        - build: macos-amd64
           os: macos-latest
```

Comment:
Pin to `macos-13`? Seems weird to change it on tests but not here.

Reply:
Sorry, I missed it.